app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -167,24 +173,35 @@ class BrowserFragment : Fragment(), BackHandler {
 
         startPostponedEnterTransition()
 
-        QuickActionComponent(
+        quickActionSheetStore = Store(
```

Comment:
I think we'd have to use `StoreProvider.get` here to get a view model see: https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/java/org/mozilla/fenix/search/SearchFragment.kt#L75

This way restoring state is "just going to work".

Reply:
@sblatz @boek ultimately, we should also try and only use one store per fragment. of course, while migrating individual components/parts of the fragment, it'll make sense to introduce separate stores. we'd just need a follow up to unify them once done. wdyt?